### DISCUSSION TITLE
Add support for larger energy and power units

### DISCUSSION THREAD

**Discussion by jschlyter** [home-assistant/architecture#1167] [Entity Models] [answered]
Reviewing the energy and power units I see that the following units are missing:

- Power: MW/GW/TW
- Energy: GWh/TWh

Adding support for these would be helpful when showing current measurements and long term statistics from the public grid. It would also align power/energy to both support all of kilo/mega/giga/tera.

**Comment by emontnemery** [comment, marked as answer]
The proposal was discussed and approved in a core meeting today 👍

**Comment by jschlyter** [comment]
PR in https://github.com/home-assistant/core/pull/130089
